velox/functions/lib/aggregates/SingleValueAccumulator.h

```diff
@@ -33,6 +33,16 @@ struct SingleValueAccumulator {
 
   bool hasValue() const;
 
+  /// Returns < 0 if 'left' is less than 'right' at 'index', 0 if
```

Comment:
Comments needs updating.

Let's remove the other 'compare' method and update the callers to use the new method. I suspect most of these callers are processing complex values with nulls incorrectly now.

Reply:
>Let's remove the other 'compare' method

@mbasmanova Should we still keep the original `SingleValueAccumulator::compare` method? Its nullHandle mode is NoStop, which is used by SparkSQL's min/max and their nullHandle mode is `NoStop`. Or remove the original `compare` method and update the callers in SparkSQL with the `NoStop` flag.

```SQL
scala> spark.sql("select min(c0) FROM VALUES array(1, 2, 3), array(null, 1) AS tab(c0)").show()
+---------+
|  min(c0)|
+---------+
|[null, 1]|
+---------+
```

Reply:
Let's remove the original 'compare' method and update the callers.

Reply:
OK